src/hooks/useZoraCreate.tsx

```diff
@@ -56,12 +68,23 @@ const useZoraCreate = () => {
         contracts: [{ ...(newParameters as any) }],
         capabilities,
       } as any)
+      setLoading(false)
     } catch (err) {
+      setLoading(false)
       console.error(err)
     }
   }
 
-  return { create, createdContract }
+  useEffect(() => {
+    if (callsStatus?.status !== 'CONFIRMED') return
+    toast({
+      title: 'Success',
+      description: 'Published On Zora Successfully!',
+      variant: 'default',
+    })
+  }, [callsStatus])
```

Comment:
this check should already exist in a different hook right?
- if no, please create shared hook to wait for tx to be confirmed